Apps/Src/Temperature.c

```diff
@@ -108,10 +143,19 @@ ErrorStatus Temperature_ChannelConfig(uint8_t tempChannel) {
 		Minions[board].com.tx_data[4] = (AUX_I2C_NO_TRANSMIT << 4) + 0xF;
 		Minions[board].com.tx_data[5] = (0xF << 4) + AUX_I2C_NACK_STOP;
     }
+	//release mutex
+  	OSMutexPost(&MinionsASIC_Mutex, OS_OPT_POST_NONE, &err);
+  	assertOSError(err);
 
     // Send data
     wakeup_sleep(NUM_MINIONS);
```

Comment:
Is the timing important on this command?

Reply:
I am not sure. I can move the OSMutexPost() to before this line to be safe

Reply:
Instead of doing that, might as well just not yield the mutex. This function call sits in between a Post() and Pend().

Reply:
Ok